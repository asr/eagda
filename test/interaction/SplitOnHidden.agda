data Nat : Set where
  zero : Nat
  suc : Nat → Nat

test : ∀{N M : Nat} → Nat → Nat → Nat
test N M = {!.N N .M!}
