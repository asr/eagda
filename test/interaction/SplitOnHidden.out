Agda2> (agda2-status-action "")
(agda2-info-action "*Type-checking*" "" nil)
(agda2-highlight-clear)
(agda2-info-action "*Type-checking*" "Checking SplitOnHidden (SplitOnHidden.agda). " t)
(agda2-info-action "*Type-checking*" "Finished SplitOnHidden. " t)
(agda2-status-action "")
(agda2-info-action "*All Goals*" "?0 : Nat " nil)
((last . 1) . (agda2-goals-action '(0)))
Agda2> Agda2> Agda2> ((last . 2) . (agda2-make-case-action '("test {zero} {zero} zero M = ?" "test {zero} {suc M} zero M₁ = ?" "test {zero} {zero} (suc N) M = ?" "test {zero} {suc M} (suc N) M₁ = ?" "test {suc N} {zero} zero M = ?" "test {suc N} {suc M} zero M₁ = ?" "test {suc N} {zero} (suc N₁) M = ?" "test {suc N} {suc M} (suc N₁) M₁ = ?")))
((last . 1) . (agda2-goals-action '(0)))
Agda2> Agda2> Agda2> (agda2-info-action "*Error*" "SplitOnHidden.agda:6,12-23 Unbound variable .x when checking that the expression ? has type Nat" nil)
((last . 3) . (agda2-maybe-goto '("SplitOnHidden.agda" . 103)))
(agda2-highlight-load-and-delete-action)
(agda2-status-action "")
Agda2> 